XMonad/Actions/WorkspaceCursors.hs

```diff
@@ -199,7 +199,7 @@ workspaceCursors :: Cursors String -> l a -> ModifiedLayout WorkspaceCursors l a
 workspaceCursors = ModifiedLayout . WorkspaceCursors
 
 newtype ChangeCursors = ChangeCursors { unWrap :: Cursors String -> X (Cursors String) }
-    deriving (Typeable)
+    
```

Comment:
Same here, these removals should not introduce extra whitespace

Reply:
Sorry, I fixed it in the latest commit, but for all files. :)